modular_zubbers/modules/clothing/code/donator_clothing.dm

```diff
@@ -0,0 +1,4 @@
+/obj/item/clothing/suit/toggle/labcoat/vic_dresscoat_donator // modified on request of nikotheguydude, the person who donated for this upstream
```

Comment:
I think this file may already exist somewhere.

Reply:
well honestly i dunno where id start looking